### PR TITLE
feat(services): add form routes

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: src/api/services/form/form.v6.service.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+angular.module('ovh-api-services').service('OvhApiServicesFormV6', ($resource) => $resource('/services/:serviceId/form/:formName', {
```
Comment (FredericEspiau) [inline src/api/services/form/form.v6.service.js]: No cache ?

Comment (marie-j) [reply src/api/services/form/form.v6.service.js]: Copy/paste from parent I guess, not entirely sure it is needed as main usage is for this is to open link from email meaning that a new app is started

Comment (FredericEspiau) [reply src/api/services/form/form.v6.service.js]: It's your usage of it, not all the future usage of it 😱

Comment (marie-j) [reply src/api/services/form/form.v6.service.js]: I guess if cache is needed at one point, it can always be added

Comment (FredericEspiau) [reply src/api/services/form/form.v6.service.js]: 😅

**Pull request merged**